origin.spec

```diff
@@ -113,6 +113,7 @@ Provides:       atomic-openshift-node
 %package clients
 Summary:        %{product_name} Client binaries for Linux
 Provides:       atomic-openshift-clients
+Obsoletes:      atomic-openshift-clients
```

Comment:
per discussion on slack this looks wrong:
https://coreos.slack.com/archives/CB95J6R4N/p1557155759054800